samples/bluetooth/peripheral_iso/README.rst

```diff
@@ -0,0 +1,24 @@
+.. _peripheral_iso:
```

Comment:
Suggest that the samples be named iso_peripheral and iso_central to go from general to specific name

Reply:
IMHO, it also makes it easier to find the "partner sample" in the folder.

Reply:
I don't disagree, but this was done to be consistent with other peripheral examples which are defined this way.